features/step_definitions/administration_steps.rb

```diff
@@ -130,3 +130,29 @@ def select_date(date, options = {})
 When(/^I change the book price to "(.*?)"$/) do |new_price|
   fill_in 'Price', with: new_price
 end
+
+When(/^I visit the admin orders url$/) do
+  visit admin_orders_path
+end
+
+Then(/^I see the orders$/) do
+  expect(page).to have_content("Total")
+  expect(page).not_to have_content("There are no Orders yet.")
+end
+
+When(/^I change the book discount to "([^"]*)"$/) do |discount|
+  fill_in 'Discount', with: discount
+end
+
+Then(/^I see the book has the discount "([^"]*)"$/) do |discount|
+  expect(page).to have_content(discount)
+end
+
+Then(/^I see the book has the discount price "([^"]*)"$/) do |discount_price|
```

Comment:
Hmm, since this and the one above it are testing the same thing, I think the wording of this one might be a typo. However, to run with it, there's a way to combine the two into one step definition.